content/using-sagemaker-managed-spot-training/notebooks/running_notebooks.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Running Notebooks"
+chapter: false
+weight: 30
+---
+
+1. Within JupyterLab (Which you opened in the previous steps), navigate to the ***/ec2-spot-labs/ec2-spot-sagemaker-managed-spot-training/*** folder.
+
+2. Double-click on a Notebook and complete each step outlined in the Notebook. Each available lab comes in the form of a separate Notebook. Locate the labs you are interested in running and complete each step documented in the notebook.  ![SageMaker Notebook Instance](/images/using-sagemaker-managed-spot-training/jupyter-3.png)
+
+3. As you are running through your labs, you can continue on to the following sections to learn more about configuring different libraries to use Managed Spot Training, as well as some best practices for reviewing acheived cost savings.
```

Comment:
I didn't really understand what the flow is here. are we just going to run in the notebook? complete it and go back to the workshop website? run then at the same time?

Reply:
The Notebooks contain the meat of the workshop. The workshop really is just instructions for getting into the Notebook. There are certain places in the notebooks where there's some idle time (ie: while waiting for the training jobs to complete). The idea was you could use that time and come back to the workshop to learn more about the frameworks and cost savings. I can re-word this a bit to make it more clear.